src/main/java/homework/home10012019/reflectionAPI/Main.java

```diff
@@ -0,0 +1,21 @@
+package homework.home10012019.reflectionAPI;
+
+import java.lang.reflect.Constructor;
+import java.lang.reflect.InvocationTargetException;
+
+public class Main {
+    public static void main(String[] args) throws ClassNotFoundException, NoSuchMethodException, InstantiationException, IllegalAccessException, InvocationTargetException {
```

Comment:
Is it ok, when main method throws exceptions? I mean, where are they handled then?

Reply:
it's a good questions!